src/legacy/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -24,19 +24,11 @@ import * as Rx from 'rxjs';
 import { filter, first } from 'rxjs/operators';
 import 'ui/vis/map/service_settings';
 import { toastNotifications } from 'ui/notify';
-import { uiModules } from 'ui/modules';
 import chrome from 'ui/chrome';
 
 const WMS_MINZOOM = 0;
 const WMS_MAXZOOM = 22;//increase this to 22. Better for WMS
 
-const emsServiceSettings = new Promise((resolve) => {
-  uiModules.get('kibana').run(($injector) => {
```

Comment:
@nreese please have a look. Not sure that we need it

Reply:
Yes, looks like that can be safely removed since serviceSettings is getting injected in the current version.